tt_eager/tt_dnn/op_library/transformer_tms/kernels/dataflow/reader_ssm_prefix_scan.cpp

```diff
@@ -4,10 +4,28 @@
 
 #include "dataflow_api.h"
 
+void fill_zeros(uint32_t cb_id) {
+    constexpr uint32_t num_zeros_reads = 2048 / MEM_ZEROS_SIZE;
```

Comment:
Should probably make 2048 a `constexpr` variable and comment that this is for `bfloat16` only. Zeros CB for other data types will be different.